### PR TITLE
feature: ZENKO-1842: azure-info-models

### PR DESCRIPTION
**Pull request by ploki**
Added Azure info models for storage accounts, containers and blobs

### CODE REVIEW

**Review by JianqinWang — APPROVED** [review]
are the param descriptions left intentionally blank or will they be filled at a later date?

**Review by philipyoo — APPROVED** [review]

**Comment by ploki** [comment]
> are the param descriptions left intentionally blank or will they be filled at a later date?

added the descriptions

**Comment by alexanderchan-scality** [comment]
minor: should the new models also be exposed through the `index.js` file?

File: lib/models/ObjectMDAzureInfo.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+     *   pending copy operation targeting this object
+     * @param {number} obj.blobSequenceNumber - sequence number for a PageBlob
+     * @param {Date} obj.blobAccessTierChangeTime - date of change of tier
+     * @param {boolean} obj.blobUncommitted - A block has been put for a
+     *   nonexistent blob which is about to be created
+     */
+    constructor(obj) {
+        this._data = {
+            containerPublicAccess: obj.containerPublicAccess,
+            containerStoredAccessPolicies: obj.containerStoredAccessPolicies,
```
Comment (alexanderchan-scality) [inline lib/models/ObjectMDAzureInfo.js:32]: was wondering if it would beneficial to validate the structure of these properties.

Comment (ploki) [reply lib/models/ObjectMDAzureInfo.js:32]: it's not worth the complexity

**Review by rahulreddy — APPROVED** [review]

**Comment by ploki** [comment]
@bert-e approve

**Pull request merged**